src/Dapr.Actors/Runtime/IActorTimer.cs

```diff
@@ -13,12 +13,6 @@ namespace Dapr.Actors.Runtime
     /// </summary>
     public interface IActorTimer
     {
-        /// <summary>
-        /// Gets the time when timer is first due.
-        /// </summary>
-        /// <value>Time as <see cref="System.TimeSpan"/> when timer is first due.</value>
-        TimeSpan DueTime { get; }
```

Comment:
Do you know what this interface is used for? (I don't). If it's not something users use, then it should be removed or non-public.